.solhint.json

```diff
@@ -1,21 +1,20 @@
 {
-    "extends": "solhint:recommended",
-    "plugins": [],
-    "rules": {
-        "compiler-version": ["error","0.8.17"],
-        "max-line-length": ["error", 200],
-        "func-visibility": ["error", {"ignoreConstructors": true}],
-        "ordering": "error",
-        "func-param-name-mixedcase": "error",
-        "modifier-name-mixedcase": "error",
-        "code-complexity": ["error",7],
-        "function-max-lines": ["error",50],
-        "no-inline-assembly": "off",
-        "no-empty-blocks": "off",
-        "max-states-count": "off",
-        "no-global-import": "off",
-        "not-rely-on-time": "off",
-        "reason-string": "off"
-    }
+  "extends": "solhint:recommended",
```

Comment:
check with team @codinghistorian @vito-kovalione the preference in styling and use the single one. better to document in confluence

Reply:
I would suggest increasing the function-max-lines limit from 50 to 150, as the FixedSpreadLiquidationStrategy.execute function is currently around 100 lines.

Other configurations look good to me, and I’m fine with them as they are.